website/blog/2025-03-27-20-must-try-mongodb-queries-operations-ferretdb.md

```diff
@@ -275,7 +275,7 @@ db.books.aggregate([
 ])
 ```
 
-Find out more about aggregation pipelines in FerretDB [here](https://docs.ferretdb.io/usage/aggregation/).
+Find out more about [aggregation pipelines in FerretDB](https://docs.ferretdb.io/usage/aggregation/).
```

Comment:
@Fashander this link is broken, should it link to `category/aggregation-operations/`?